app/src/main/java/com/internship/move/feature/onboarding/OnboardingViewModel.kt

```diff
@@ -0,0 +1,17 @@
+package com.internship.move.feature.onboarding
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.internship.move.model.OnBoardingRepository
+import kotlinx.coroutines.launch
+
+class OnboardingViewModel(private val repo: OnBoardingRepository) : ViewModel() {
+
+    suspend fun getLoggedStatus() = repo.getLoggedStatus()
+
+    fun changeLogStatus(logValue: Boolean) = viewModelScope.launch { repo.changeLoggedStatus(logValue) }
+
+    suspend fun getAuthStatus(): String? {
```

Comment:
It's a bit strange to require a status and get a string.

Usually, when you mean `Status`, you have a limited set of options. By returning a string, you get an unlimited set of options.

I believe you want to see if the user's logged in or not. For this, better return a boolean and do the checks inside the VM